modules/notifications/notifications.php

```diff
@@ -217,19 +193,33 @@ public function get_notification_footer( $post ) {
 	 * @param string $message_headers. (optional) Message headers
 	 */
 	public function send_email( $action, $post, $subject, $message, $message_headers = '' ) {
+		// Do nothing if the post isn't set.
+		$post_id = $post->ID;
+		if ( ! $post_id ) {
+			return;
+		}
 
-		// Get list of email recipients -- set them CC
-		$recipients = $this->get_notification_recipients( $post, true );
-
-		if ( $recipients && ! is_array( $recipients ) ) {
-			$recipients = explode( ',', $recipients );
+		// Ensure the email address is set from settings.
+		if ( empty( VIP_Workflow::instance()->settings->module->options->email_address ) ) {
+			return;
 		}
 
-		// ToDo: Figure out best filters to add for the email.
+		$email_recipients[] = VIP_Workflow::instance()->settings->module->options->email_address;
```

Comment:
This sort of pattern is usually used when appending to an already-existing array. I'd suggest instantiating the variable here as an array to be more explicit:

```suggestion
		$email_recipients = [ VIP_Workflow::instance()->settings->module->options->email_address ];
```